vllm/_custom_ops.py

```diff
@@ -9,6 +9,8 @@
 
 logger = init_logger(__name__)
 
+use_awq_triton = False
```

Comment:
Is there a plan to set this variable?

Reply:
Not sure of this, maybe there is a better place, like env or something? I will ask for a suggestion, or maybe you have one.  I will also look.

Reply:
OK, I added a new env for it

Reply:
@tlrmchlsmth I closed https://github.com/vllm-project/vllm/pull/6850, so this PR will be the sole AWQ Triton PR for my work, there is another possibly coming for KV cache and Flash attention.  So, the awq_dequantize_kernel is now for this PR too.